infra/recipes/docker-compose/common/aws-services.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.3"
+services:
+  minioS3:
+    image: minio/minio
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+    ports:
+      - 9871:9001
+      - 9870:9000
```

Comment:
What is the use of this port mapping? It seems `9871` is minio ui port and that is mapped to 9001.

Reply:
yes, 9001 is where the minio UI get spinned up to and we map it to 9871 of our local machine. We need this mapping.